configuration/drugs/drugs.csv

```diff
@@ -55,7 +55,6 @@ ff094ae9-f707-4a4c-befc-e64de5195172,,Metronidazole 250mg,Cambodia_Metronidazole
 88209b1e-5b32-4f39-a5b2-acaaa31ae0ac,,Omeprazole 20mg,Cambodia_Omeprazole,Capsule,20mg,
 165c9b15-5cc7-444d-be87-52ff2c4ccd26,,Oral rehydration 20.5g,Cambodia_Oral rehydration,Suspension,20.5g,
 18d992b2-846c-4ad3-9af8-bbf06b4d6142,,Paracetamol 500mg,Cambodia_Paracetamol,Tablet,500mg,
-a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e,,Paracetamol 200mg,Cambodia_Paracetamol,Tablet,200mg,

```

Comment:
@ouiliam in order to void this drug could you use the Void/Retire field directly at this line, in this file?
I mean I see that in "drugs_void.csv" you have added this drug's UUID so what you have done is correct. Just that the "drugs_void.csv" is more for the initial voiding of the default data (that come with the demo).
In this particular case, the drug we void is one of our durgs, let's use the Void/Retire field in this file for that.

Reply:
@rbuisson OK, have amended the changes, got this error but normal as drug already voided:

ERROR - CsvParser.saveAll(172) |2018-03-19 11:53:23,715| An OpenMRS object could not be constructed or saved from the following CSV line: [a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e, TRUE, Paracetamol 200mg, Cambodia_Paracetamol, Tablet, 200mg, null]
org.openmrs.api.ValidationException: 'Drug[hashCode=eec0d862,uuid=a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e]' failed to validate with reason: concept: An empty concept is not allowed
WARN - HibernateConceptDAO.getConceptByName(1741) |2018-03-19 11:53:23,784| Multiple concepts found for 'Tablet'